packages/database/prisma/migrations/20230921110737_initial/migration.sql

```diff
@@ -0,0 +1,271 @@
+-- CreateTable
+CREATE TABLE `Study` (
+    `id` INTEGER NOT NULL,
+    `name` VARCHAR(191) NOT NULL,
+    `status` VARCHAR(191) NOT NULL,
+    `recordStatus` VARCHAR(191) NOT NULL,
+    `route` VARCHAR(191) NOT NULL,
+    `irasId` VARCHAR(191) NULL,
+    `protocolReferenceNumber` VARCHAR(191) NULL,
+    `sampleSize` INTEGER NULL,
+    `chiefInvestigatorFirstName` VARCHAR(191) NULL,
+    `chiefInvestigatorLastName` VARCHAR(191) NULL,
+    `managingSpeciality` VARCHAR(191) NOT NULL,
+    `plannedOpeningDate` DATETIME(3) NULL,
+    `plannedClosureDate` DATETIME(3) NULL,
+    `actualOpeningDate` DATETIME(3) NULL,
+    `actualClosureDate` DATETIME(3) NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `Organisation` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) NOT NULL,
+    `rtsIdentifier` VARCHAR(191) NOT NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    UNIQUE INDEX `Organisation_name_key`(`name`),
```

Comment:
Not sure we should have a unique index on name

Reply:
I needed a unique field for the query to prevent duplicates, perhaps it would have been better to use the rts identifier?

Reply:
Yes, RTS identifier is probably best.. is this because of the upsert operation we're using?